components/src/core/user-menu/user-menu.tsx

```diff
@@ -98,6 +98,7 @@ export const UserMenu: React.FC<UserMenuProps> = (props) => {
                                 iconColor={iconColor || menuItem.iconColor}
                                 fontColor={fontColor || menuItem.fontColor}
                                 backgroundColor={backgroundColor || menuItem.backgroundColor}
+                                dense
```

Comment:
I just merged this, but actually we need to merge this with what's in the menuItem too so someone can override it if they want:

```
dense={menuItem.dense !== undefined ? menuItem.dense : true}
```

Reply:
The reason why this is a thing is because we are spreading the props from the user's configuration (`menuItem`) on line 92 as the first props on the ILI. So if they specify a prop and we specify a default value, the effect is like this:
```
<InfoListItem
    propName={userValue}
    ...
    propName={ourDefaultValue}
/>
```
and the last one in wins. So our default value would take effect even if the end user tries to override it in their configuration object.

Reply:
Another way to do this would be to move the spread after our default props, but that can be a little harder to see when you are scanning through code:
```
<InfoListItem
    key={index}
    iconColor={iconColor}
    fontColor={fontColor}
    backgroundColor={backgroundColor}
    dense
    {...menuItem}
    onPress={(): void => {
        closeMenu();
        if (menuItem.onPress) menuItem.onPress();
    }}
    styles={Object.assign(menuItemStyles, {
        title: Object.assign(
            {
                fontSize: 16,
                fontFamily: theme.fonts.regular.fontFamily,
                fontWeight: theme.fonts.regular.fontWeight,
            },
            menuItemStyles.title
        ),
    })}
/>
```